selfhosting/auto-scaling-aws.md

```diff
@@ -0,0 +1,91 @@
+# AWS EKS auto scaling instructions
+
+Unlinke GCP, AWS EKS doesn't enable autoscaling by default. You need to install
+[autoscaler][k8s-autoscaler] specific to AWS EKS cluster. Thanks to helm, we
+can install cluster-autoscaler chart. First prepare iam roles and node groups
+as suggested in readme of [helm chart][helm-cluster-autoscaler]. Steps to
+follow:
+
+
```

Comment:
Should we also add a link to our set up eks instructions ?

Reply:
Makes sense. Wondering how that would be in gitbooks?

Reply:
Will have to manually update the link in git book. That’s fine

Reply:
>Will have to manually update the link in git book. That’s fine

Which is this link, you guys are talking about?

Reply:
https://docs.getdockup.com/how-tos/set-up-dockup-agent-on-aws